src/RageSoundReader_Resample_Good.cpp

```diff
@@ -680,7 +680,7 @@ int RageSoundReader_Resample_Good::Read( float *pBuf, int iFrames )
 
 	{
 		int iFramesNeeded = m_apResamplers[0]->NumInputsForOutputSamples(iFrames);
-		float *pTmpBuf = (float *) alloca( iFramesNeeded * sizeof(float) * iChannels );
+		float pTmpBuf[iFramesNeeded * sizeof(float) * iChannels];
```

Comment:
I think only gcc allows sizing the array this way.  Other compilers like clang do not allow it.